polypomo

```diff
@@ -133,6 +138,57 @@ class Status:
             self.timer = Timer(self.worktime)
 
 
+@contextmanager
+def setup_conn():
+    global DATABASE_PATH
+    # check if the database exist
+    if (os.path.exists(DATABASE_PATH)):
```

Comment:
This if clause can be removed if the SQL query below is converted to:
`CREATE TABLE IF NOT EXISTS`